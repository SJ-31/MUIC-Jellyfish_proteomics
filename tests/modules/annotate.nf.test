nextflow_process {

    name "Test Process ANNOTATE"
    script "modules/annotate.nf"
    process "ANNOTATE"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
                input[0] = Channel.fromPath("/mnt/data/shannc/nf/results/jellyfish/1-First_pass/Unmatched/BLAST/db_hits-nO_D.tsv")
                input[1] = "/mnt/data/shannc/nf/tests/modules"
                """
            }
        }

        then {
            assert process.success
        }

    }

}
