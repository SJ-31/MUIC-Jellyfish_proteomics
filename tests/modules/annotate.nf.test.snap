{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    [
                        "nO_D_anno.tsv:md5,7b126220c696c807d30295e4ee0588f4",
                        "nO_D_meta.tsv:md5,375c0fcc10cd3f613b0b38298e90e07c"
                    ]
                ],
                "1": [
                    
                ],
                "annotations": [
                    [
                        "nO_D_anno.tsv:md5,7b126220c696c807d30295e4ee0588f4",
                        "nO_D_meta.tsv:md5,375c0fcc10cd3f613b0b38298e90e07c"
                    ]
                ]
            }
        ],
        "timestamp": "2023-12-25T11:50:34.298201"
    }
}